ui-app/client/src/components/FileUpload/FileUpload.tsx

```diff
@@ -159,9 +161,25 @@ function FileUpload(props: ComponentProps) {
 		);
 	}, [bindingPathPath]);
 
-	const setFiles = (files: any) => {
+	const setFiles = async (files: any) => {
 		if (uploadType === 'FILE_OBJECT') {
-			setData(bindingPathPath!, isMultiple ? [...files] : files[0], context?.pageName);
+			let stringFiles = [];
```

Comment:
Hey, not like this, you need to add another upload type, binary file to string. In line 191 switch the statement you need to add this type.